src/main/java/org/testng/TestRunner.java

```diff
@@ -1483,7 +1461,7 @@ public static int getVerbose() {
     return m_verbose;
   }
 
-  public void setVerbose(int n) {
+  public static void setVerbose(int n) {
```

Comment:
I don't intend to but if you search the usage.. there are 10+ places that use a static method TestRunner.getVerbose()